docs/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Solaar
+layout: default
+---
+
+**Solaar** is a Linux device manager for Logitech's [Unifying][unifying] Receiver
+peripherals. It is able to pair/unpair devices to the receiver, and for most
+devices read battery status.
+
+It comes in two flavors, command-line and GUI.  Both are able to list the
+devices paired to a Unifying Receiver, show detailed info for each device, and
+also pair/unpair supported devices with the receiver.
+
+[unifying]: http://logitech.com/promotions/6072
+
+## Supported Devices
+
+**Solaar** will detect all devices paired with your Unifying Receiver, and at
+the very least display some basic information about them.
+
+For some devices, extra settings (usually not available through the standard
+Linux system configuration) are supported. For a full list of supported devices
+and their features, see [docs/devices.md]({{ site.baseurl }}/devices).
+
+
+## Pre-built packages
+
+Pre-built packages are available for a few Linux distros.
+
+* Debian 7 (Wheezy) or higher: packages in this [repository]({{ site.baseurl }}/debian)
+* Ubuntu/Kubuntu 16.04+: use the `solaar-gnome3` and/or `solaar` package from [universe repository][universe repository]
+* Ubuntu/Kubuntu 12.04 to 14.04: use `solaar` package from [ppa:daniel.pavel/solaar][ppa]
+
+The `solaar` package uses a standard system tray implementation; to ensure
+integration with *gnome-shell* or *Unity*, install `solaar-gnome3`.
+
+* an [Arch package][arch], courtesy of Arnaud Taffanel
+* a [Fedora package][fedora], courtesy of Eric Smith
+* a [Gentoo package][gentoo], courtesy of Carlos Silva and Tim Harder
+* a [Mageia package][mageia], courtesy of David Geiger
+* an [OpenSUSE rpm][opensuse], courtesy of Mathias Homann
+
+[ppa]: http://launchpad.net/~daniel.pavel/+archive/solaar
+[arch]: http://aur.archlinux.org/packages/solaar
+[fedora]: https://apps.fedoraproject.org/packages/solaar
+[gentoo]: https://packages.gentoo.org/packages/app-misc/solaar
+[mageia]: http://mageia.madb.org/package/show/release/cauldron/application/0/name/solaar
+[opensuse]: http://software.opensuse.org/package/Solaar
+[universe repository]: http://packages.ubuntu.com/search?keywords=solaar&searchon=names&suite=all&section=all
```

Comment:
How much of this is still up-to-date? Should the README be copied over here? Is it possible to symlink or reduce duplication is some way?

Reply:
The README has been updated since I've last worked on this PR, so I'll update it. Currently, I don't know of any way we could reduce duplication of this, because of the front matter which is required at the top of the file. A compromise would be to add that to the repo's readme.

Reply:
It looks like Github supports symlinks in the other direction (README.md -> docs/README.md). I'll merge this and try that. Thanks!